typographie/admin.py

```diff
@@ -18,7 +18,13 @@ def apply_typographie_to_dict(d, fields):
         elif isinstance(v, list):
             new_dict[k] = []
             for i in v:
-                new_dict[k].append(apply_typographie_to_dict(i, fields))
+                if isinstance(i, str):
+                    if k in fields:
+                        new_dict[k].append(typographie(i))
+                    else:
+                        new_dict[k].append(i)
```

Comment:
We could do as a few lines below.

```suggestion
                    if k in fields:
                        i = typographie(i)
                    new_dict[k].append(i)
```

Reply:
Agreed and done. Sorry i already rebased.